src/internal_macros.rs

```diff
@@ -299,7 +280,7 @@ macro_rules! display_from_debug {
 macro_rules! hex_script (($s:expr) => (::blockdata::script::Script::from(::hex::decode($s).unwrap())));
 
 #[cfg(test)]
-macro_rules! hex_hash (($s:expr) => (::util::hash::Sha256dHash::from(&::hex::decode($s).unwrap()[..])));
+macro_rules! hex_hash (($s:expr) => (::bitcoin_hashes::Hash::from_slice(&::hex::decode($s).unwrap()).unwrap()));
```

Comment:
You can replace the bitcoin_hashes::Hash with bitcoin_hashes::sha256d::Hash::from_slice here to make it match the old version.